lib/inc/facter/facts/fact.hpp

```diff
@@ -434,11 +434,15 @@ namespace facter { namespace facts {
         /**
          * The fact for virtualization hypervisor.
          */
-        constexpr static char const* virtualization = "virtual";
+        constexpr static char const* virt = "virtual";
```

Comment:
Considering this is just a constant defined for internal usage, I think either name is fine, doesn't matter IMO.